.github/workflows/emscripten.yml

```diff
@@ -0,0 +1,62 @@
+name: Test Emscripten/Pyodide build
+
+on:
+  push:
+    branches:
+      - emscripten-ci
+  pull_request:
+    branches:
+      - main
+      - maintenance/**
+
+jobs:
+  build-wasm-emscripten:
+    # run this for now on all CI for now in case something starts breaking it
+    # if: "success() && (startsWith(github.ref, 'refs/tags/') || github.ref == 'refs/heads/main')"
+    # needs: [test, lint]
```

Comment:
This needs a `if: github.repository == 'numpy/numpy'` condition just like all the other actions have - preventing it from running on forks by default. Although this action does not trigger on a push to `main`, so perhaps it can be left out here.